account_payment_order/models/account_move_line.py

```diff
@@ -23,6 +26,25 @@ class AccountMoveLine(models.Model):
         string="Payment lines",
     )
 
+    @api.depends(
+        "move_id", "move_id.invoice_partner_bank_id", "move_id.payment_mode_id"
```

Comment:
Isn't the partner_bank_id redundant now that Odoo added an invoice_partner_bank_id on the move level?

Reply:
I don't think so, I explain one situation:
You have to pay your employees, so you will create an "account move" to generate this movement. Once you have to pay it, you want to associate the "account.move.line" to a payment order, so you have to be able to associate a "bank account" to an "account.move.line".

Reply:
Yes, good point.